core/App/navigators/defaultStackOptions.ts

```diff
@@ -1,18 +1,22 @@
 import { Theme } from '../theme'
 
-export function createDefaultStackOptions({ ColorPallet }: Theme) {
+export function createDefaultStackOptions(theme: Theme) {
```

Comment:
@wadeking98 Why was the change needed? I think the previous method of deconstructing was okish. Just asking - not a value statement.

Reply:
I needed to access properties outside of the ColourPallete like the textTheme

Reply:
Perhaps just add `TextTheme` to the destructuring assignment. Will also make it clear you only need those fields.

Reply:
change completed